java-strings/src/main/java/com/baeldung/string/SubstringPalindrome.java

```diff
@@ -0,0 +1,90 @@
+package com.baeldung.string;
+
+import java.util.HashSet;
+import java.util.Set;
+
+public class SubstringPalindrome {
+
+    public Set<String> isPalindromeUsingCenter(String input) {
```

Comment:
When a method name begins with "is", one expects it to return boolean. A better method name would be _findAllPalindromesUsingCenter_.

Reply:
Changed named